tests/react/abortable.test.tsx

```diff
@@ -42,21 +43,27 @@ describe('abortable atom test', () => {
     )
 
     await findByText('loading')
-    resolve.splice(0).forEach((fn) => fn())
-    await findByText('count: 0')
+
+    act(() => {
+      resolve.splice(0).forEach((fn) => fn())
+    })
+    await waitFor(() => expect(getByText('count: 0')).toBeTruthy())
```

Comment:
Can this be reverted to findByText too? I think it's the same as we `waitFor` anyway.

Reply:
Good call! [Done](https://github.com/pmndrs/jotai/pull/1773#:~:text=test%3A%20use%20findByText%20over%20getByText)